src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -72,13 +72,20 @@ export default {
       type: Object,
       required: false,
     },
+    standardColorIdentifier: {
+      type: String,
+      required: false,
+    },
   },
   computed: {
     // get the alias, if any, and fallback to the `teal` color
     color: ({ type }) => HeroColorsMap[TopicTypeAliases[type] || type] || HeroColors.teal,
-    styles: ({ color }) => ({
+    styles: ({ color, standardColorIdentifier }) => ({
       // use the color or fallback to the gray secondary, if not defined.
-      '--accent-color': `var(--color-documentation-intro-accent, var(--color-type-icon-${color}, var(--color-figure-gray-secondary)))`,
+      '--accent-color': `var(--color-documentation-intro-accent, var(--color-type-icon-${color}))`,
+      // if a `standardColor` is provided, first try the more specific variant,
+      // falling back to the generic color variant.
+      '--standard-accent-color': standardColorIdentifier && `var(--color-standard-${standardColorIdentifier}-documentation-intro-fill, var(--color-standard-${standardColorIdentifier}))`,
```

Comment:
I'm just curious, because I don't know the answer off hand—does Vue ignore "falsy" attributes of `styles` when injecting it into the actual DOM element?

Reply:
From what I saw yes hehe